packages/f/font-droid-ttf/files/droid.metainfo.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2023 Solus Developers <copyright@getsol.us> -->
+<component type="font">
+  <id>droid</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <name>Droid</name>
```

Comment:
Please add a homepage url for the font in the form of

```
<url type="homepage">https://fontwebpage.com/</url>
```

Reply:
there is no homepage for this font, i tried to find it. unless dafont or font-squirrel counts as homepage

Reply:
Fair point. I also looked, found this font set was deprecated by the Noto family, and that Google stopped publishing a web page for it.